release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2712,3 +2712,24 @@ link:https://access.redhat.com/solutions/6843421[{product-title} 4.9.26 containe
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-27"]
+=== RHSA-2022:1158 - {product-title} 4.9.27 bug fix and security update
+
+Issued: 2022-04-06
+
+{product-title} release 4.9.27, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:1158[RHSA-2022:1158] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1157[RHBA-2022:1157] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6885691[{product-title} 4.9.27 container image list]
+
+[id="ocp-4-9-27-bug-fixes"]
+==== Bug fixes
+
+* Previously, a bug in Cisco's ACI caused a failed error when running new Virtual Machines (VMs) in OpenStack. This fix adds additional filters in the OpenStack cluster-API-provider, ensuring VMs run properly with Cisco ACI. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2064633[*BZ#2064633*])
```

Comment:
The wording here is mostly just a suggestion. OpenStack should be in {rh-openstack-first} and  {rh-openstack} per our [contributor guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-name-and-version) on common attributes.

```suggestion
* Previously, a bug in Cisco's ACI resulted in an error when running new virtual machines in {rh-openstack-first}. With this update, additional filters were added in the {rh-openstack} cluster-API-provider. Virtual machines now run properly with Cisco ACI. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2064633[*BZ#2064633*])
```

Reply:
Done! :)